### ISSUE TITLE
Chrissy hates her blog layout -- i agree

### ISSUE THREAD

**Issue by tbbooher** [tbbooher/ptp_nova2#49]
look at: http://www.ptpnova.com/blog/ -- it could be better.

**Comment by bezierer** [comment]
Yeah—gets kind of janky on mobile, too. I've actually got some good ideas for it. I'll work on some “design in the browser” on my localhost and push up some changes when I've got it looking good.

**Comment by momof4** [comment]
When would be a good time to chat?
I'm around this afternoon and this evening.

Thanks,
Chrissy

On Thu, May 28, 2015 at 9:27 AM, Cameron Campbell <notifications@github.com>
wrote:

> Yeah—gets kind of janky on mobile, too. I've actually got some good ideas
> for it. I'll work on some “design in the browser” on my localhost and push
> up some changes when I've got it looking good.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tbbooher/ptp_nova2/issues/49#issuecomment-106308682>.
>



-- 

chrissy@theboohers.org

chrissy@ptpnova.com

571-212-6145 (C)

**Comment by tbbooher** [comment]
cool -- glad to see this is moving forward.

**Comment by tbbooher** [comment]
all -- just want to make sure the momentum stays high.

tim

On Thu, May 28, 2015 at 11:26 AM, momof4 <notifications@github.com> wrote:

> When would be a good time to chat?
> I'm around this afternoon and this evening.
>
> Thanks,
> Chrissy
>
> On Thu, May 28, 2015 at 9:27 AM, Cameron Campbell <
> notifications@github.com>
> wrote:
>
>
> > Yeah—gets kind of janky on mobile, too. I've actually got some good ideas
> > for it. I'll work on some “design in the browser” on my localhost and
> push
> > up some changes when I've got it looking good.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/tbbooher/ptp_nova2/issues/49#issuecomment-106308682
> >.
> >
>
>
>
> --
>
> chrissy@theboohers.org
>
> chrissy@ptpnova.com
>
> 571-212-6145 (C)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tbbooher/ptp_nova2/issues/49#issuecomment-106408781>.
>



-- 
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Tim Booher
www.theboohers.org | tim@theboohers.org
703.649.4264 (home)
(703) 822-7863 (mobile)
